caculate percent commission/main.js

```diff
@@ -0,0 +1,13 @@
+calPerCommission = (sales) => {
```

Comment:
Bắt buộc phải dùng từ khóa khai báo
```suggestion
const calPerCommission = sales => {
```
Hàm có duy nhất 1 param thì có thể bỏ `()`